operations/mimir-mixin/dashboards/ruler.libsonnet

```diff
@@ -42,8 +42,29 @@ local filename = 'mimir-ruler.json';
         ),
       )
       .addPanel(
-        $.panel('Write to ingesters - QPS') +
-        $.statPanel('sum(rate(cortex_ingester_client_request_duration_seconds_count{%s, operation="/cortex.Ingester/Push"}[$__rate_interval]))' % $.jobMatcher($._config.job_names.ruler), format='reqps')
+        $.panel(
+          if $._config.show_ingest_storage_panels then
+            'Writes to ingesters / ingest storage - RPS'
+          else
+            'Writes to ingesters - RPS'
+        ) +
+        $.statPanel(
```

Comment:
How is this going to work with a mix of cells, using both architectures? Dashboard can't dynamically change. I think we need to include both panels :(

Reply:
Exactly.